### ISSUE TITLE
Need better check to confirm whether browser_args are an Array...

### ISSUE THREAD

**Issue by the-t-in-rtf** [testem/testem#1119]
When using the `testem.js` form of the configuration file, I have been constructing my own browser_args before exposing the options via `module.exports`.  I was seeing strange behavior where only single strings were accepted as browser args, and arrays were being silently ignored.

When I turned on debug logging in Testem, I saw that my arguments were being ignored with messages like:

```
WARN Type error: Chrome's "args" property should be a string or an array
```

Looking at the code in a debugger, it turns out the `Array` of browser arguments I'm exposing to Testem is not passing the `instanceof Array` tests  in `browser-args.js`, for example, this one:

https://github.com/testem/testem/blob/241b1e42e96cd7e4d656efda6475e96e1f7978ae/lib/utils/browser-args.js#L39

To better detect actual Arrays in a wider range of circumstances, it seems like we'd be better off using the more modern `Array.isArray`, [as discussed here](http://web.mit.edu/jwalden/www/isArray.html).

In this case, in the debugger, I can confirm that `Array.isArray` check works for my use case (generating options in a `testem.js` file and then exposing them to Testem via `module.exports`).

Unless there are other preferred ways to achieve the same goal, I am happy to submit a pull replacing all `instanceof Array` checks with `Array.isArray` checks.

**Comment by johanneswuerbach** [comment]
Do you know which node versions support `Array.isArray`? Testem supports 0.10+ at this point, but I was planning to release a breaking v2 which only supports 4+

**Comment by the-t-in-rtf** [comment]
@johanneswuerbach, good question, I will check that shortly.

**Comment by the-t-in-rtf** [comment]
Looks like 0.10.0 does support `Array.isArray`:

```
Tonys-MacBook-Pro:testem duhrer$ node --version
v0.10.48
Tonys-MacBook-Pro:testem duhrer$ node
> Array.isArray([]);
true
> 
```

**Comment by johanneswuerbach** [comment]
This has been released as `v1.16.1`

**Comment by the-t-in-rtf** [comment]
Thanks, @johanneswuerbach!
